pkg/workflow_template.go

```diff
@@ -295,6 +295,21 @@ func (c *Client) getWorkflowTemplate(namespace, uid string, version int64) (work
 
 	workflowTemplate.Labels = labelsMap[workflowTemplate.WorkflowTemplateVersionID]
 
+	params, err := workflowTemplate.GetParametersKeyString()
```

Comment:
Is this supposed to be here or in a different PR?

Reply:
Different PR, I branched off the wrong branch. Closing this PR, use this one instead. https://github.com/onepanelio/core/pull/425